lib/voxpupuli/acceptance/serverspec_extensions.rb

```diff
@@ -0,0 +1,12 @@
+require 'serverspec'
+require_relative 'serverspec_extensions/curl_command'
+
+module Serverspec
+  module Helper
+    module Type
+      def curl_command(*args)
+        Voxpupuli::Acceptance::ServerspecExtensions::CurlCommand.new(*args)
+      end
+    end
+  end
+end
```

Comment:
Technically, we don't need to pee into the `Serverspec` namespace, just doing
```
        define_method 'curl_command' do |*args|
          eval "Voxpupuli::Acceptance::ServerspecExtensions::CurlCommand.new(*args)"
        end
```
or similar is enough